src/activitypub/fediverse-bridge.ts

```diff
@@ -95,7 +110,7 @@ export class FediverseBridge {
                 preview,
                 url: post.url,
                 to: PUBLIC_COLLECTION,
-                cc: post.author.apFollowers,
+                ccs: ccs,
```

Comment:
Curious about the use of `ccs` instead of `cc`

I can only find docs for `cc` (https://www.w3.org/TR/activitypub/#create-activity-outbox, https://www.w3.org/TR/activitystreams-vocabulary/#dfn-cc) and that seems to be what Mastodon & co also use?

Reply:
While creating the FedifyNote we can't use cc to add multiple URLs or Objects

This is how it's defined in constructor of Note Object :-

```
cc?: Object | URL | null;
ccs?: (Object | URL)[];
```

However, I know what you mean, when you look at an activity you'll see the array 'cc'. This is because Fedify internally converts ccs -> cc

I have created [this activity](https://princis-macbook-pro-2.tail5da2a.ts.net/.ghost/activitypub/note/07444f77-12fb-46f1-8af6-78fe899fc512) with my local code. I added all recipients in 'ccs' but in the activity you'll see 'cc' (similar to Mastodon)

Reply:
Ahhh it's a Fedify thing - sorry! Got confused